server/application-server/src/main/resources/application-prod.yml

```
@@ -52,7 +52,7 @@ hephaestus:
       sender: ${MAIL_SENDER:hephaestus@aet.cit.tum.de}
 
     detection:
-        automatic-detection-enabled: ${AUTOMATIC_DETECTION_ENABLED:false}
+        automatic-detection-enabled: ${AUTOMATIC_DETECTION_ENABLED:true}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)